include/cuco/detail/probing_scheme_impl.inl

```diff
@@ -107,7 +107,7 @@ __host__ __device__ constexpr auto linear_probing<CGSize, Hash>::operator()(
 {
   using size_type = typename Extent::value_type;
   return detail::probing_iterator<Extent>{
-    cuco::detail::sanitize_hash<size_type>(hash_(probe_key)) % upper_bound,
+    probing_scheme_base_type::template sanitize_hash<size_type>(hash_(probe_key)) % upper_bound,
```

Comment:
This could be simplified a bit, e.g.
```cpp
struct linear_probing {
  ...
  using probing_scheme_base_type::sanitize_hash;
  ...
};
```
Then here 
```suggestion
    sanitize_hash<size_type>(hash_(probe_key)) % upper_bound,
```
should be working just fine.

Reply:
sanitize_hash being a template, we still need it :).